dev/README.md

```diff
@@ -0,0 +1,13 @@
+To generate developer version `cd` to the root of this repo, then run:
```

Comment:
The Makefile should be updated to have a rule that does this, and these instructions should simply say something like "cd to the root of this repo and type `make dev`" or something.

Reply:
Given that we're using Bikeshed, which is Python based, I'd rather not add a nodejs dependency.